projects/opendr_ws/src/perception/scripts/rgbd_hand_gesture_recognition.py

```diff
@@ -95,37 +99,58 @@ def callback(self, image_data, depth_data):
         ros_gesture = self.bridge.from_category_to_rosclass(gesture_class)
         self.gesture_publisher.publish(ros_gesture)
 
-    def preprocess(self, image, depth_img):
-        '''
-        Preprocess image, depth_image and concatenate them
-        :param image_data: input image
-        :type image_data: engine.data.Image
+    def preprocess(self, rgb_image, depth_img):
+        """
+        Preprocess rgb_image, depth_image and concatenate them
+        :param rgb_image: input RGB image
+        :type rgb_image: engine.data.Image
         :param depth_data: input depth image
         :type depth_data: engine.data.Image
```

Comment:
Docstring doesn't match argument
```suggestion
        :param depth_img: input depth image
        :type depth_img: engine.data.Image
```